client/src/action/dos/deleteFileForCounty.ts

```diff
@@ -0,0 +1,38 @@
+import action from 'corla/action';
+import { endpoint } from 'corla/config';
+import { empty } from 'corla/util';
+
+const deleteFileUrl = endpoint('delete-file');
+
+async function deleteFileForCounty(fileType: string, countyId: number) {
+    const init: RequestInit = {
+        body: JSON.stringify( { fileType, countyId } ),
+        credentials: 'include',
+        method: 'post',
+    };
+
+    try {
+        action('DELETE_FILE_SEND', { fileType, countyId });
+
+        const r = await fetch(deleteFileUrl, init);
+
+        const received = await r.json().catch(empty);
+
+        if (!r.ok) {
+            action('DELETE_FILE_FAIL', {fileType, received});
+            return false;
+        }
+
+        action('DELETE_FILE_OK', {fileType, countyId, received});
```

Comment:
Since you almost certainly want to avoid the page refresh requirement here, the last step is adding a reducer tied to this event that modifies the state to indicate the file has been uploaded, like this one: https://github.com/democracyworks/ColoradoRLA/blob/master/client/src/reducer/root.ts#L113-L121

Reply:
There were some complications there, but I came up with something that works.